agents/loco_mc_agent.py

```diff
@@ -330,8 +323,8 @@ def perceive(self, force=False):
             # Send data to the dashboard timeline
             hook_data = {
                 "name" : "perceive",
+                "datetime" : datetime.datetime.now(),
```

Comment:
We should record both start and end time of the event and show it in timeline.
This is also supported/reflected in visjs --> event items will have different length based on duration: https://visjs.github.io/vis-timeline/examples/timeline/groups/visibleGroups.html

Reply:
One question I have about this is that the project plan states "the time is discrete, which means you can only select a time point when an activity occurred." This is also shown in the wireframes. I can add it right now but just to be clear, the project plan specs have changed?

Reply:
Our conclusion from offiline discussion: the timeline can either be in physical time mode where events have duration (start time & end time corresponded to the physical time) or in logical time mode where time is discrete and events occurs at discrete time point.
For now we are using physical time mode, but it's a good-to-have thing to be able to switch time mode -- putting it to stretch goal.